package.json

```diff
@@ -424,6 +425,7 @@
     "base64-js": "^1.3.1",
     "bitmap-sdf": "^1.0.3",
     "brace": "0.11.1",
+    "byte-size": "^8.1.0",
```

Comment:
Mind adding a section for this to https://github.com/elastic/kibana/blob/main/renovate.json so updates can be assigned to your team?

Reply:
And want to confirm there were no other existing dependencies that provided this functionality.

Reply:
Added. let me know if the entry looks correct. TY

Reply:
oh, we have another lib xtermjs that wasn't added to this file. will include as part of our entry.

Reply:
Great, thanks!